cd/application-deployment/dev/vaec-api-task-definition.json

```
@@ -327,8 +327,8 @@
   "requiresCompatibilities": [
     "FARGATE"
   ],
-  "cpu": "512",
-  "memory": "1024",
+  "cpu": "1024",
+  "memory": "4096",
```

```suggestion
  "memory": "2048",
```
Sorry, was still looking this one up to make sure we could drop down to 2 Gigs